packages/clients/src/aptos/endpoints.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2023-2024 Shinami Corp.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Region, createRegionalApiUrl } from "../region.js";
+
+export const NodeIndexerUrls: Record<Extract<Region, "us1">, string> = {
```

Comment:
Let's also add this file to `aptos/index.ts` like what you did in `sui/index.ts`.